test/resources/reftest-analyzer.js

```diff
@@ -228,10 +228,15 @@ window.onload = function () {
         });
         continue;
       }
-      match = line.match(/^ {2}IMAGE[^:]*: (.*)$/);
+      match = line.match(/^ {2}IMAGE[^:]*\((\d+)x(\d+)x(\d)+\): (.*)$/);
```

Comment:
Is there a typo the last digit group, i.e. did you mean the following instead?
```suggestion
      match = line.match(/^ {2}IMAGE[^:]*\((\d+)x(\d+)x(\d+)\): (.*)$/);
```